Formula/sc-im.rb

```diff
@@ -0,0 +1,24 @@
+class ScIm < Formula
+  desc "Spreadsheet program for the terminal, using ncurses"
+  homepage "https://github.com/andmarti1424/sc-im"
+  url "https://github.com/andmarti1424/sc-im/archive/v0.6.0.tar.gz"
+  sha256 "5da644d380ab3752de283b83cce18c3ba12b068d0762c44193c34367a0dcbc38"
+  head "https://github.com/andmarti1424/sc-im.git", :branch => "freeze"
+
+  depends_on "ncurses"
+
+  def install
+    cd "src" do
+      system "make", "prefix=#{prefix}", "LDFLAGS=-L#{Formula["ncurses"].lib}"
```

Comment:
I think it's also more common to use `ENV.prepend` to modify build flags; see e.g. [`burp.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/burp.rb)